### PR TITLE
Revised Taxonomy

### PR DESCRIPTION
**Pull request by morchickit**
This is a revised taxonomy with Amitay where we added some branches and consolidated others.

### CODE REVIEW

File: taxonomy.simple.yaml [hunk 1]
```diff
@@ -433,24 +357,23 @@
...
-    - Developmental Disability
     - Intellectual Disability
     - Learning Disability:
       - ADD/ADHD
     - Mental Illness
     - Limited Mobility
     - Physical Disability
     - Visual Impairment
+    - Mentally debilitated
+    - In need for nursing care
```
Comment (akariv) [inline taxonomy.simple.yaml:370]: Capitalize

File: taxonomy.simple.yaml [hunk 2]
```diff
@@ -54,167 +43,118 @@
...
   - Education:
     - Help Find School
-    - Help Pay for School:
-      - Books
-      - Clothes for School
-      - Financial Aid & Loans
-      - Supplies for School
-      - Transportation for School
+    - Education Institutions:
+      - Nurserie
```
[the remark this answers is not in the document]
Comment (morchickit) [reply taxonomy.simple.yaml:69]: Yup, fixed

File: taxonomy.simple.yaml [hunk 3]
```diff
@@ -54,167 +43,118 @@
...
     - Emergency Helplines
-    - Emergency Payments:
-      - Help Pay for Food
-      - Help Pay for Healthcare
-      - Help Pay for Housing
-      - Help Pay for School
     - Emergency Shelter
     - Help Find Missing Persons
-    - Immediate Safety:
-      - Help Escape Violence
```
[the remark this answers is not in the document]
Comment (morchickit) [reply taxonomy.simple.yaml:136]: We swapped it in a different branch

File: taxonomy.simple.yaml [hunk 4]
```diff
@@ -54,167 +43,118 @@
         - Parenting Education
```
[the remark this answers is not in the document]
Comment (morchickit) [reply taxonomy.simple.yaml:43]: That was Amitay call to have this service just once.

File: taxonomy.simple.yaml [hunk 5]
```diff
@@ -226,37 +166,31 @@
         - Family Counseling
         - Group Therapy
         - Individual Counseling
-        - Substance Abuse Counseling
-      - Hospital Treatment
       - Medications for Mental Health
       - Mental Health Evaluation
-      - Outpatient Treatment:
+      - hospitalization:
```
[the remark this answers is not in the document]
Comment (morchickit) [reply taxonomy.simple.yaml:171]: Done

File: taxonomy.simple.yaml [hunk 6]
```diff
@@ -295,88 +225,81 @@
...
         - Safe Housing
-      - Short-Term Housing:
-        - Old Age Home
-        - Sober Living
-        - Vacations
-    - Temporary Shelter
-  
+      - Short-Term Housing
+    - Temporary Shelter:
+      - Shelters from violence
```
[the remark this answers is not in the document]
Comment (morchickit) [reply taxonomy.simple.yaml:230]: Done

File: taxonomy.simple.yaml [hunk 7]
```diff
@@ -295,88 +225,81 @@
...
+      - Transit flats
```
[the remark this answers is not in the document]
Comment (morchickit) [reply taxonomy.simple.yaml:231]: Done

**Comment by morchickit** [comment]
@akariv - I made changes, take a look

File: taxonomy.simple.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (akariv) [reply taxonomy.simple.yaml:43]: I understand, it's just that this instance feels more esoteric than the others.

File: taxonomy.simple.yaml [hunk 8]
```diff
@@ -1,6 +1,7 @@
 - Human Services:
   - Care:
     - Accessibility
+      - Translation
```
Comment (akariv) [inline taxonomy.simple.yaml:4]: This is too indented...

File: taxonomy.simple.yaml [hunk 9]
```diff
@@ -9,37 +10,25 @@
...
-        - Help Escape Violence
-        - Safe Housing
-        - Safety Education
-    - Residential Care:
-      - Assisted Living
-      - Nursing Home
+    - Residential Care
     - Recovery:
       - Vacations
+      - Vocational rehabilitation
```
Comment (akariv) [inline taxonomy.simple.yaml:31]: Capitalize

File: taxonomy.simple.yaml [hunk 10]
```diff
@@ -54,209 +43,154 @@
...
+        - University Prep
       - Tutoring
       - Youth Development
       - Youth Movement
     - Leadership development
-    - Preschool
     - Screening & Exams:
-      - English as a Second Language (ESL)
       - GED/High-School Equivalency
+    - English as a Second Language (ESL)
```
Comment (akariv) [inline taxonomy.simple.yaml:102]: Duplicate

File: taxonomy.simple.yaml [hunk 11]
```diff
@@ -54,167 +43,118 @@
...
-      - Parenting Education
       - Sex Education
-    - Help Pay for Healthcare:
-      - Disability Benefits
-      - Discounted Healthcare
-      - Health Insurance
-      - Medical Supplies
-      - Prescription Assistance
-      - Transportation for Healthcare
-    - Medical Care:
```
[the remark this answers is not in the document]
Comment (akariv) [reply taxonomy.simple.yaml:206]: This entire section changed indentation - was this on purpose? I think it makes sense to keep the 'medical care' hierarchy.

File: taxonomy.simple.yaml [hunk 12]
```diff
@@ -295,88 +225,82 @@
...
     - Special Employment:
       - Supported Employment
       - Protected Employment
       - Protected Factory
     - Workplace Rights
+  
+  - Culture: 
+    - Fine Art
+    - Cinema: 
+      - Exhibitions
```
Comment (akariv) [inline taxonomy.simple.yaml:289]: Is this supposed to be indented here?

**Review by akariv — CHANGES_REQUESTED** [review]

File: taxonomy.simple.yaml (hunk 8 above)
Comment (morchickit) [reply taxonomy.simple.yaml:4]: It's not, it was missing :

File: taxonomy.simple.yaml (hunk 10 above)
Comment (morchickit) [reply taxonomy.simple.yaml:102]: Not a duplicated actually but needed to be deleted non the less. Good catch.

File: taxonomy.simple.yaml (hunk 12 above)
Comment (morchickit) [reply taxonomy.simple.yaml:289]: No, fixed.

**Comment by morchickit** [comment]
I added - medical care to the taxonomy but I remember Amitay wanted that out

**Review by akariv — CHANGES_REQUESTED** [review]

File: taxonomy.simple.yaml [hunk 13]
```diff
@@ -483,28 +407,6 @@
       - Nursing Care
     - Neoplasm:
       - Cancer:
```
Comment (akariv) [inline taxonomy.simple.yaml:409]: Remove the ':'

File: taxonomy.simple.yaml [hunk 14]
```diff
@@ -54,209 +43,154 @@
...
-        - Checkup & Test
-        - Disability Screening
         - Disease Screening
         - Eye Tests
         - Genetic Testing
         - Hearing Tests
         - Pregnancy Tests
-      - Combined Care
-      - Maternity Care
+        - Maternity Care
```
Comment (akariv) [inline taxonomy.simple.yaml:158]: Maternity care was indented by mistake I think (it wasn't inside checkup & test before)

File: taxonomy.simple.yaml [hunk 15]
```diff
@@ -9,37 +10,25 @@
...
-    - Guidance 
-    - Navigating the System:
-      - Help Fill out Forms
-      - Help Find Housing
-      - Case Management
-      - Help Find Childcare
-      - Help Find School
-      - Help Find Work
+    - Guidance
+    - Eligibility checks:
```
Comment (akariv) [inline taxonomy.simple.yaml:18]: Capitalize

File: taxonomy.simple.yaml [hunk 16]
```diff
@@ -54,209 +43,154 @@
...
         - Hearing Tests
         - Pregnancy Tests
-      - Combined Care
-      - Maternity Care
+        - Maternity Care
       - Medical Supplies:
         - Assistive Technology
         - Prosthesis
     - Mental Health Care:
-      - Anger Management
```
[the remark this answers is not in the document]
Comment (morchickit) [reply taxonomy.simple.yaml:223]: Fixed

**Comment by morchickit** [comment]
Fixed

**Review by akariv — APPROVED** [review]

**Pull request merged**